examples/fmsynth/mmlplayer_score.h

```diff
@@ -0,0 +1,58 @@
+/****************************************************************************
+ * apps/examples/fmsynth/mmlplayer_score.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __EXAMPLES_FMSYNTH_MMLPLAYER_SCORE_H
+#define __EXAMPLES_FMSYNTH_MMLPLAYER_SCORE_H
+
+static const char *floh_walzer_right =
```

Comment:
Where this is used?

Reply:
Could you please answer this?

Reply:
I'm asking because placing this in header file and usage of `static` and if header is included into multiple C files, then a separate copy of data will be created so size will be multiple of number of inclusions. To prevent this the best case is to remove `static` and make a symbol global that will cause an error at linking stage. Or leave a place for a "human factor" and keep current approach.